src/scripts/devstart.sh

```diff
@@ -6,9 +6,16 @@
 # * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # *
 
+extra_args=
+if [ -f /.dockerenv ]; then
+	# This is passed to vite, which exposts the port to the
+	# host (outside the container)
+	extra_args="--host"
+fi
+
```

Comment:
This wasn't needed with before because the Webpack development server automatically listens on the public network interface (not just loopback).